specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -1461,6 +1461,11 @@
           "type": "boolean",
           "description": "The flag to signal HBI data in the workspace and reduce diagnostic data collected by the service",
           "default": false
+        },
+        "serviceProvisionedResourceGroup": {
+          "type": "string",
+          "description": "The name of the scoped resource group created by workspace RP in customer subscription",
```

Comment:
Change to "The name of the scoped resource group created by workspace RP in customer subscription if the workspace is cmk workspace"?